src/test/java/seedu/address/model/item/field/GithubTest.java

```diff
@@ -0,0 +1,64 @@
+package seedu.address.model.item.field;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class GithubTest {
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new Github(null));
+    }
+
+    @Test
+    public void constructor_invalidGitHub_throwsIllegalArgumentException() {
+        String invalidGithub = "";
+        assertThrows(IllegalArgumentException.class, () -> new Github(invalidGithub));
+    }
+
+    @Test
+    public void isValidGithub() {
+        // null github
+        assertThrows(NullPointerException.class, () -> Github.isValidGithub(null));
```

Comment:
Put this method in an `isValidGithub_nullInput_throwsNullPointerException()` method.